azext_edge/edge/_help.py

```diff
@@ -1058,211 +1013,108 @@ def load_iotops_help():
-            \\\"session\\\": {\\\"timeout\\\": 60000}, \\\"subscription\\\": {\\\"maxItems\\\": 1000}, \\\"security\\\": {
-            \\\"autoAcceptUntrustedServerCertificates\\\": true}}"
-        - name: Create an asset endpoint with username-password user authentication and additional configuration with prefilled values
-                (bash syntax example).
-          text: >
-            az iot ops asset endpoint create --name myAssetEndpoint -g myRG --cluster myCluster
-            --target-address "opc.tcp://opcplc-000000:50000" --username-ref "aio-opc-ua-broker-user-authentication/opc-plc-username"
-            --password-ref "aio-opc-ua-broker-user-authentication/opc-plc-password"
-            --additional-config '{"applicationName": "opcua-connector", "defaults": {
-            "publishingIntervalMilliseconds": 100,  "samplingIntervalMilliseconds": 500,  "queueSize": 15,},
-            "session": {"timeout": 60000}, "subscription": {"maxItems": 1000}, "security": {
-            "autoAcceptUntrustedServerCertificates": true}}'
+        - name: Create an asset endpoint with anonymous user authentication using the given instance in the same resource group.
+          text: >
+            az iot ops asset endpoint create opcua --name myprofile -g myresourcegroup --instance myinstance
+            --target-address opc.tcp://opcplc-000000:50000
+        - name: Create an asset endpoint with anonymous user authentication using the given instance in a different resource group and subscription.
+          text: >
+            az iot ops asset endpoint create opcua --name myprofile -g myresourcegroup --instance myinstance
+            --instance-resource-group myinstanceresourcegroup --instance-subscription myinstancesubscription
+            --target-address opc.tcp://opcplc-000000:50000
+        - name: Create an asset endpoint with username-password user authentication using the given instance in the same resource group.
+          text: >
+            az iot ops asset endpoint create opcua --name myprofile -g myresourcegroup --instance myinstance
+            --target-address opc.tcp://opcplc-000000:50000
+            --username-ref myusername --password-ref mypassword
+        - name: Create an asset endpoint with certificate user authentication using the given given instance in the same resource group.
+          text: >
+            az iot ops asset endpoint create opcua --name myprofile -g myresourcegroup --instance myinstance
+            --target-address opc.tcp://opcplc-000000:50000 --certificate-ref mycertificate.pem
+        - name: Create an asset endpoint with anonymous user authentication and recommended values for the OPCUA configuration using the given instance in the same resource group.
+                Note that for successfully using the connector, you will need to have the OPC PLC service deployed on your cluster and the target address must point to the service.
```

Comment:
The opc ua connector can connect to an opc ua server outside the cluster. This makes it sound like an opc ua server needs to be co-located on the cluster.